samples/bot-ai-meeting-helper/.localConfigs1

```diff
@@ -0,0 +1,22 @@
+BOT_ID=
```

Comment:
Remove it from gitIgnore and push without renaming.
Hari has also followed so refer his PR:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/1300#pullrequestreview-2203548900

Reply:
Done